### ISSUE TITLE
What happens to theta transferred to my guardian nodes public address?

### ISSUE THREAD

**Issue by tekforce1701** [thetatoken/guardian-mainnet-guide#86]
What happens to theta transferred to my guardian nodes public address?

**Comment by Garitty7612** [comment]
You mistakenly did that ?

**Comment by tekforce1701** [comment]
yes 
    On Friday, March 19, 2021, 04:13:32 AM MST, David Garry ***@***.***> wrote:  
 
 


You mistakenly did that ?

**Comment by Garitty7612** [comment]
Wow. Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by tekforce1701** [comment]
It says i have a balance of 237 theta and 0 Tfuel
What happens to the theta balance ?
    On Friday, March 19, 2021, 04:13:32 AM MST, David Garry ***@***.***> wrote:  
 
 


You mistakenly did that ?

**Comment by raptor742** [comment]
> You mistakenly did that ?

Do you or anyone else have any info on this? I sent 1 theta a while back as a test to my first guardian node. Is there anyway for it to send it back to the original sender? This seems like a HUGE issue that I am very sure other's have had issues with. Guardian nodes shouldn't even be able to receive tokens in this way, imo.

Thanks for any updates if anyone has heard about this before.

**Comment by Garitty7612** [comment]
Yeah I’ve had about this. You can contact Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by Garitty7612** [comment]
@raptor742 Yeah I’ve had about this. You can contact Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by raptor742** [comment]
> You mistakenly did that ?

So, in case no one saw, that telegram link you left here led to a fake theta group. I pity all the people getting scamed by those people in there to enter then passwords on the sites they are linking to. Did you know it was fake? Are you part of the group scamming people? Or did you not know that it was a fake theta group?

**Comment by raptor742** [comment]
OK, now this is getting bad. I do not know how to use github well, and do not know how to message any of the moderators/admins of this project properly. Could one of you guys please leave a message here detailing what is going on? Deleting that guys messages is probably good, but I am still completely in the dark. Thank you.

**Comment by tekforce1701** [comment]
I sorriusy did not know
    On Sunday, March 21, 2021, 12:24:02 PM MST, raptor742 ***@***.***> wrote:  
 
 



You mistakenly did that ?


So, in case no one saw, that telegram link you left here led to a fake theta group. I pity all the people getting scamed by those people in there to enter then passwords on the sites they are linking to. Did you know it was fake? Are you part of the group scamming people? Or did you not know that it was a fake theta group?

**Comment by raptor742** [comment]
> I sorriusy did not know On Sunday, March 21, 2021, 12:24:02 PM MST, raptor742 ***@***.***> wrote: You mistakenly did that ? So, in case no one saw, that telegram link you left here led to a fake theta group. I pity all the people getting scamed by those people in there to enter then passwords on the sites they are linking to. Did you know it was fake? Are you part of the group scamming people? Or did you not know that it was a fake theta group? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

You did not leave a bad link. That guy who had originally replied to you did leave a bad link. You haven't done anything wrong here on github. Just hopefully you did not go to any sites that the guy had shown you. Don't give your passwords or mnemonic phrases AND DONT ENTER YOUR MENMONIC PHRASE ANYWHERE (especially links that anyone sends you.)
